src/MEstimators.jl

```diff
@@ -172,30 +173,32 @@ Construct with syntax:
 
 Unfilled ranges will automatically use the standard L2-estimator.
 """
-immutable MultiEstimator{Estimators<:Tuple,Ranges<:Tuple} <: MEstimator
+struct MultiEstimator{Estimators<:Tuple,Ranges<:Tuple} <: MEstimator
     est::Estimators
     rng::Ranges
 
-    function MultiEstimator(est::Estimators,rng::Ranges)
-        if length(est) != length(rng)
-            error("Must have same number of estimators and ranges")
-        end
-        for i = 1:length(est)
-            if !isa(est[i],MEstimator)
-                error("Expecting an M-estimator, got $(est[i]) of type $(typeof(est[i]))")
-            end
-            if !method_exists(Base.getindex,(Vector{Float64},typeof(rng[i])))
-                error("Expecting an indexible range, got $(rng[i]) of type $(typeof(rng[i]))")
-            end
-        end
-        new(est,rng)
+    function MultiEstimator{Estimators, Ranges}(est::Estimators, rng::Ranges) where {Estimators<:Tuple, Ranges<:Tuple}
+    	if length(est) != length(rng)
```

Comment:
Something weird has happened with indentation here. Perhaps your editor is set to use tabs?

Reply:
:+1: